init.rb

```diff
@@ -48,6 +48,18 @@
 
 current_flavor = Rock.flavors.current_flavor
 
+#The following option is used to determine which xml importer should be used.
+#Currently castxml is working
+
+configuration_option('TYPELIB_CXX_LOADER', 'string',
+    :default => 'castxml',
+    :possible_values => ['gccxml', 'castxml'],
+    :doc => [
+        "Which xml importer should be used?",
+        "Currently only castxml is working when using master flavor"])
```

Comment:
This is not true. It is only required on gcc5 machines. One way could be to try and detect whether gcc is gcc5 and select gccxml/castxml on this basis.